pallets/runtime/develop/src/fee_details.rs

```diff
@@ -36,6 +36,8 @@ enum CallType {
     AcceptMultiSigSigner,
     AcceptIdentitySigner,
     AcceptIdentityPrimary,
+    /// Matches any authorization available for `auth.authorized_by` payer redirection.
+    Any,
```

Comment:
Perhaps a more descriptive name would be more suitable? `AuthorizedBy`?

Reply:
Well `CallType` is a filter on the type of "authorization call", and `::Any` means it doesn't apply any additional restrictions, which is accurate. Meanwhile, `CallType::AuthorizedBy` doesn't suggest what types of calls it accepts.

Reply:
I mean that `CallType::Any` would mean "any" call type to a layman reader which would include all other call types (elements in this enum).

From the way it is used in the code, the new enum variant is just another call type like any of `Accept` variants, and it is not some set of call types. Which is why `Any` is not the right name.

This is especially so because you have to match on
```
CallType::Any | CallType::AcceptIdentitySigner
```
in one place. If `Any` included all other call types you wouldn't have to specify the other variant.

Please feel free to find any other name that wouldn't be as generic as it is now.

Reply:
Maybe `AcceptAuthorization` is a good alternative?

Reply:
Looking again at the code, it looks like this CallType is really "What is the user trying to do with the authorisation, based on the extrinsic they've called?". We then match this against the type of authorisation the user has provided, check, and apply logic as necessary.